controller/proxy-injector/webhook.go

```diff
@@ -79,6 +78,23 @@ func Inject(api *k8s.API,
 	}
 	proxyInjectionAdmissionRequests.With(admissionRequestLabels(ownerKind, request.Namespace, report.InjectAnnotationAt, configLabels)).Inc()
 
+	//Check if Namespace has valid inject annotation
+	if isNsAnnotationValid, val := isInjectionValid(nsAnnotations); !isNsAnnotationValid {
+		log.Warnf("invalid proxy injection value detected at namespace %s - \"%s\"", request.Namespace, val)
+		if parent != nil {
+			recorder.Eventf(*parent, v1.EventTypeWarning, eventTypeSkipped, "Invalid proxy injection detected at namespace : %s:%s", pkgK8s.ProxyInjectAnnotation, val)
+		}
+	}
+
+	//Check if Pod-template has valid inject annotation
+	podAnnotations := resourceConfig.GetPodAnnotations()
+	if isPodAnnotationValid, val := isInjectionValid(podAnnotations); !isPodAnnotationValid {
+		log.Warnf("invalid proxy injection value detected at pod-template %s - \"%s\"", report.ResName(), val)
```

Comment:
Should we also log the workload type and name. i.e `invalid injection annotation value detected at namespace/pod/name`? 
That information would be useful.

Reply:
@Pothulapati with this approach, the workload type and name is being logged already because of `report.ResName` - eg; `invalid proxy injection value detected at pod-template  pod/pod-name"`. We could go ahead and prepend the namespace as well

Reply:
That's awesome.
It would be cool, to have the log message be in the same format at ns or workload level then like `invalid proxy injection annotation value detected at <type>/<name>"`